prems-resources/clinical-data/src/main/resources/SLING-INF/content/apps/cards/mailTemplates/bodyTemplate.header.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html>
+  <head>
+    <title>UHN Canadian Patient Experience Survey</title>
```

Comment:
Since we're using the same title for all emails, let's make it more generic.
```suggestion
    <title>UHN Patient Experience Survey</title>
```